rpc/send.go

```diff
@@ -32,6 +32,11 @@ import (
 	"github.com/gogo/protobuf/proto"
 )
 
+// TODO(pmattis): This code is only used by kv.DistSender. Perhaps move it to
+// the kv package. We might also want to de-generalize it so that it knows
```

Comment:
I think so, but I'd hold off on doing that work until the first grpc code is in.